src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -187,10 +187,26 @@ private IList<AnnotatedCodeLocation> NormalizeRawMessage(string rawMessage, out
                     tokens = location.Split(':');
 
                     relatedLocations = relatedLocations ?? new List<AnnotatedCodeLocation>();
+                    PhysicalLocation physicalLocation;
 
-                    var relatedLocation = new AnnotatedCodeLocation
+                    if (tokens[0].Equals("file", StringComparison.OrdinalIgnoreCase))
+                    {
+                        // special case for file paths
```

Comment:
gotcha. created a new var 'locationTokens'. will defer the CSV parsing, we'll do that later.

---
In reply to: [134355305](https://github.com/Microsoft/sarif-sdk/pull/591#discussion_r134355305) [](ancestors = 134355305)